lib/common/repository/api_repository.dart

```diff
@@ -130,6 +130,26 @@ class ApiRepository {
         '/events/$_eventSlug/bookmark_schedule/$sessionId',
       );
 
+      return response['feedback'] as String;
+    } catch (e) {
+      rethrow;
+    }
+  }
```

Comment:
This section of code should remain as it was. Probably some oversight. Should remain as:

```dart
  return response['message'] as String;
    } catch (e) {
      rethrow;
    }
```

This way, the bookmarking functionality doesn't break.

Reply:
This was a mistaken addition apologies